package.json

```diff
@@ -73,5 +81,14 @@
   },
   "files": [
     "dist/"
-  ]
+  ],
+  "config": {
+    "commitizen": {
+      "path": "./node_modules/cz-conventional-changelog"
+    }
+  },
+  "eik": {
+    "server": "https://assets.finn.no",
+    "files": "./dist"
```

Comment:
Did we test that we need to specify the files here if the property already exists in line 83?

Reply:
I want to remember that "files" are not inherited. Only `name` and `version`.

Reply:
Can be wrong ofc 😅

Reply:
I guess it's needed only in case we don't want the same as in the base `files` entry...? Haven't looked how it is set up in the other repos... but I think I like to have it there always (for consistency and clarity)... What do you think?

Reply:
We kept it in release-playground... so I guess that was for some reason? 😄

Reply:
As I've mentioned previously we need `files` as it is only `name` and `version` from package.json that will be used if omitted in the eik config. If you remove files from eik you'll get 👇 
![image](https://user-images.githubusercontent.com/37986637/226271375-36614639-09ce-4e42-89c3-fdd91a5c658b.png)